common/src/thrift/tachyon.thrift

```diff
@@ -109,10 +109,18 @@ exception FileAlreadyExistException {
   1: string message
 }
 
+exception BlockAlreadyExistsException {
```

Comment:
I think we should put the exceptions into a separate thrift file, keep them alphabetically sorted, and add a comment that explains their purpose? It is fine to do this in a separate PR, just add a TODO (and a JIRA ticket) for this. Thanks!